rewrite-maven/src/main/java/org/openrewrite/maven/tree/MavenRepository.java

```diff
@@ -63,6 +63,8 @@ public class MavenRepository implements Serializable {
     @Nullable
     String snapshots;
 
+    @EqualsAndHashCode.Include
+    @With
     @NonFinal
```

Comment:
I'd expect the `@With` to imply `@NonFinal`, and don't see that on other fields that have `@With` in this class.
```suggestion
```

Reply:
The annotation is required because Lombok considers it as a required constructor argument if it's not present. Changing the field to `Boolean` (capital B) would be a better (and bigger) fix.